sdk/docs/sharable/sdk/quickstart/observe/observability-troubleshooting-overview.rst

```diff
@@ -204,35 +185,28 @@ backend-service you will see the lines:
       - "5055:5005"
 
 
-This enables remote debugging of the java component backend in the user
-application (backend-service). You can use this to connect an IDE
-Debugger to the service at runtime if required. Keep in mind that we
-recommend your first resort be Grafana and the consolidated logs in
-Loki, as this ensures the system remains debugable in production.
+This enables remote debugging of the java component backend in the user application (backend-service). 
+You can use this to connect an IDE Debugger to the service at runtime if required. 
+Keep in mind that we recommend your first resort be Grafana and the consolidated logs in Loki. 
+This keeps the system debugable in production.
 
 Observability and tracing
 =========================
 
 Faulty distributed systems can be notoriously hard to diagnose.
-Quickstart provides, at the start of a project, the sort of
-observability and diagnostics facilities often only developed toward the
-end of the project. Simplifying diagnostics for new Canton Network
-Applications from the outset of each project is one of the motivations
-behind the development of Quickstart.
-
-The links in the overview include the official user and reference
-documentation for the various tools included in Quickstart. While there
-is no substitute for the official documentation, it is hoped the
-following tour of the capabilities configured into Quickstart can
-provide a starting point for your own experimentation.
+Quickstart provides, at the start of a project, 
+the sort of observability and diagnostics facilities often only developed toward the end of the project. 
+Simplifying diagnostics for new Canton Network Applications from the outset of each project is one of the motivations behind the development of Quickstart.
```

Comment:
```suggestion
From the start of a project, Quickstart provides the sort of observability and diagnostics facilities that are otherwise often only developed toward the end. 
Simplifying diagnostics for new Canton Network Applications from the outset of each project is one of the motivations behind the development of Quickstart.
```